### PR TITLE
Automatic-Module-Name clause added to jar manifest.

### LINKED ISSUE #355: Automatic module name
Hi,

Would you consider adding an Automatic-Module-Name clause to the jar manifest?

```groovy
manifest {
     attributes(
        'Automatic-Module-Name': 'io.nats.jnats',
    )
}
```

Thanks.

### PR DESCRIPTION
**Pull request by bjorndarri**
Resolves #355

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -99,6 +99,9 @@ task bundle(type: Bundle) {
 }
 
 jar {
+    manifest {
+        attributes('Automatic-Module-Name': 'io.nats.jnats')
+    }
```
Comment (scottf) [inline build.gradle:104]: In the articles I found about this subject, the majority of them show this code:

```
jar {
    manifest {
        name = "mylibrary"
        instruction "Automatic-Module-Name", "io.nats.jnats"
    }
}
```

**Comment by scottf** [comment]
This is a Java 9 and later feature. It won't hurt to have this in there though.

File: build.gradle (hunk 1 above)
Comment (bjorndarri) [reply build.gradle:104]: I'll admit that I'm not much of a Maven person, so my proposed solution may not be the most correct one, would you like me to edit the PR or do you guys want to deal with this?

**Review by ColinSullivan1 — APPROVED** [review]
LGTM.  Thanks @bjorndarri!

**Pull request merged**